src/uct/ib/mlx5/ib_mlx5.h

```diff
@@ -408,6 +412,12 @@ typedef struct uct_ib_mlx5_md {
     uint8_t                  max_rd_atomic_dc;
     uint8_t                  log_max_dci_stream_channels;
     uint32_t                 smkey_index;
+    struct {
+        /* Max dp ordering level per transport, 
+           as listed in uct_ib_mlx5_dp_ordering_t */
+        uint8_t              rc;
```

Comment:
use `uct_ib_mlx5_dp_ordering_t` for clarity?

Reply:
The issue is that in those kinds of structs, we want the struct size to be as small as possible, and enum type take up to 64 bits.

Reply:
I assume we don't have size issue for MD structure as we don't have that many instances (and enum is probably 1 byte).